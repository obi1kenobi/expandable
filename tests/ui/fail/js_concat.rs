#[expandable::expr]
macro_rules! js_concat {
    ($left:expr, $right:expr) => {
        $left ++ $right
    };
}