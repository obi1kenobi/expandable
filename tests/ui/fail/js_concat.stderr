error: Potentially invalid expansion. Expected an identifier.
 --> tests/ui/fail/js_concat.rs:4:16
  |
4 |         $left ++ $right
  |                ^

warning: unused macro definition: `js_concat`
 --> tests/ui/fail/js_concat.rs:2:14
  |
2 | macro_rules! js_concat {
  |              ^^^^^^^^^
  |
  = note: `#[warn(unused_macros)]` on by default

error[E0601]: `main` function not found in crate `$CRATE`
 --> tests/ui/fail/js_concat.rs:6:2
  |
6 | }
  |  ^ consider adding a `main` function to `$DIR/tests/ui/fail/js_concat.rs`
